v0.0.4 Beta 1A

enkei64 opened the pull request:
Start the grind on v0.0.4

enkei64 committed:
Fix inspector row height and hide layer ID column

enkei64 committed:
Use new sliders

enkeii64 commented:
- Aspect Ratio lock when holding shift resizing layers with one of their corner handles
- Opacity slider for adjusting opacity
- Layers have colors now

enkei64 committed:
GUI update

enkei64 committed:
Seperate position and bound boxes

enkei64 committed:
Seperate commit and bound boxes

enkei64 committed:
Revert "Seperate commit and bound boxes"

This reverts commit 065f9b6bd9d9ffcbce425996181fbabea850cac2.

enkei64 committed:
geometryFlipped fix

enkei64 committed:
Load images in assets folder

enkei64 committed:
reset app

enkei64 committed:
better restart

enkei64 committed:
anchor point fix!!!!

enkei64 committed:
Add layer theme fix, language not updating immediately

enkei64 committed:
scale updates properly

enkei64 committed:
Handles easier to drag

enkei64 committed:
text layer fix

enkei64 committed:
Correct indentation .caml files

enkei64 committed:
Discord RPC

enkei64 committed:
Appearance and UI tab merged

enkei64 committed:
Position and bound handles fix (finally)

enkeii64 commented:
- Added Discord Rich Presence
- Merged the appearance tab with the UI tab in settings window
- Fixed position not updating when moving layer (due to layer being deselected after dragging)
- Bound handles for resizing layers now syncs with the cursor (same speed and position)

enkei64 committed:
Remove open file button, new message when nothing is selected in the inspector

enkeii64 commented:
- Removed open file button as welcome screen replaces it - RIP
- Message in the inspector when no layer is selected

enkei64 committed:
Keyboard shortcuts update

retronbv committed:
fix bug found by glixed

holy crap retronbv working on openposter??? (bug: https://discord.com/channels/1357766178000474192/1395183687846596638)

Co-Authored-By: Squair. <squairyt@gmail.com>

enkei64 committed:
New icon files

enkei64 committed:
README improvement

enkei64 committed:
README fix

enkei64 committed:
README new image

enkei64 committed:
Merge branch 'main' into v0.0.4-B1A

ItMe12s reviewed (approved):
lgtm bro, merge it 😀  hopefully not vibecoded /j

retronbv commented:
tuff
